nexus/src/external_api/http_entrypoints.rs

```diff
@@ -4382,6 +4383,25 @@ async fn rack_view(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// List uninitialized sleds in a given rack
+#[endpoint {
+    method = GET,
+    path = "/v1/system/hardware/uninitialized-sleds",
+    tags = ["system/hardware"]
+}]
+async fn uninitialized_sled_list(
+    rqctx: RequestContext<Arc<ServerContext>>,
+) -> Result<HttpResponseOk<Vec<UninitializedSled>>, HttpError> {
```

Comment:
Should we return a paginated response here? I realise the number of unitialised sleds probably won't be enough to warrant pagination, but it seems like an unwritten convention to always return a `ResultsPage<T>` with "list" endpoints.

Reply:
That was my original take as well @karencfv. But @davepacheco and I talked about it, and it seems like overkill given the physical limitations.